calyx-py/test/correctness/fifo.py

```diff
@@ -36,20 +36,20 @@ def insert_fifo(prog, name):
     len = fifo.reg("len", 32)  # The length of the FIFO.
 
     # Cells and groups to compute equality
-    cmd_eq_0 = fifo.eq_use(cmd, 0, 2)
-    cmd_eq_1 = fifo.eq_use(cmd, 1, 2)
-    cmd_eq_2 = fifo.eq_use(cmd, 2, 2)
+    cmd_eq_0 = fifo.eq_use(cmd, 0)
+    cmd_eq_1 = fifo.eq_use(cmd, 1)
+    cmd_eq_2 = fifo.eq_use(cmd, 2)
```

Comment:
Some progress here: we can now infer widths when one of the two operands in an input or output port of the component. I'd like to do the more general case (e.g., just below, we'll check if `len.out` equals 10, where `len` is a register. I can't infer that width yet...) but just pushing this for now!